scripts/start-services-no-ui.sh

```diff
@@ -62,8 +62,7 @@ fi
 
 pushd "${API_DIR}"
 printf "\n\nBuilding/starting API components...\n\n"
-# TODO: uncomment the below when we have any API dependencies to run
-#docker-compose up -d
+docker-compose up -d
```

Comment:
will this run the API itself as a container? In which case should we remove the gradle command below?

Reply:
🤦 yep - let me fix